platform/darwin/src/MGLTypes.h

```diff
@@ -18,6 +18,12 @@
 #define MGLColor NSColor
 #endif
 
+#define TARGET_OS_MACOS (TARGET_OS_MAC && !(TARGET_OS_IOS || TARGET_OS_TV || TARGET_OS_WATCH))
```

Comment:
Let’s stick to a simple `!TARGET_OS_IPHONE` check at each call site for now. By the time we introduce support for tvOS and/or watchOS, we’ll’ve dropped support for macOS 10.10 and will be able to use the standard `TARGET_OS_OSX` macro anyways.